src/robusta/core/model/nodes.py

```diff
@@ -0,0 +1,94 @@
+from pydantic.main import BaseModel
+from typing import List, Dict
+
+
+class NodeInfo(BaseModel):
+    name: str
+    node_creation_time: str
+    internal_ip: str
+    external_ip: str
+    taints: str
+    conditions: str
+    memory_capacity_mb: int
+    allocatable_memory_mb: int
```

Comment:
This is a little nit-picky but I would prefer names like `memory_allocatable` and `memory_allocated` etc such that everything related to memory starts with the prefix `memory_`. I think its more readable and works better in autocomplete.

Same for CPU

Reply:
I don't see any reason to change that

Reply:
No problem, keep it as is.

Reply:
updated it